src/renderer/shared/core/types/utility.ts

```diff
@@ -1,3 +1,7 @@
 export type PartialBy<T, K extends keyof T> = Omit<T, K> & Pick<Partial<T>, K>;
 
+export type KeysOfType<T, TCondition> = {
```

Comment:
Handy utility type
It grabs all properties from an object that satisfies condition 
Example:
```javascript
  KeysOfType<AbstractAccount, string> ==> 'name' | 'type'
```

Reply:
Nice!